src/process.cr

```diff
@@ -35,7 +35,7 @@ class Process
     LibC.getppid
   end
 
-  # Sends a *signal* to the processes identified by the given *pids*.
+  @[Deprecated("Use #signal instead")]
   def self.kill(signal : Signal, *pids : Int)
```

Comment:
The implementation of this should be in terms of `Process.signal`

Reply:
And please keep the documentation comment until the method is entirely removed.